media/css/firefox/mobile.scss

```diff
@@ -106,12 +119,43 @@ a.button.button-focus {
     }
 }
 
-html[dir="rtl"] .main-header {
-    @media #{$mq-desktop} {
-        text-align: right;
+@media #{$mq-desktop} {
+    .header-content {
+        display: table-row;
+    }
+
+    .header-title,
+    .header-info {
+        display: table-cell;
+        width: 50%;
+    }
+
+    .header-title {
+        padding-right: $header-product-tablet-padding / 2;
+        white-space: nowrap;
+    }
+
+    .header-info {
+        padding-bottom: 0.75em; // keeps bottom of text from going below header-title
```

Comment:
Will this perform as intended when all the strings are localized and have radically variable lengths?

Reply:
It working as intended so I think I need to re-word the comment ;)